config/areas/users/views.php

```diff
@@ -31,30 +33,20 @@
 					},
 					'roles' => array_values($roles),
 					'users' => function () use ($kirby, $role) {
-						$users = $kirby->users();
+						$users = new UsersCollector(
+							role: $role,
+							sortBy: 'username asc',
+							page: $kirby->request()->get('page', 1),
+							limit: 20
+						);
 
-						if (empty($role) === false) {
-							$users = $users->role($role);
-						}
-
-						// sort users alphabetically
-						$users = $users->sortBy('username', 'asc');
-
-						// paginate
-						$users = $users->paginate([
-							'limit' => 20,
-							'page'  => $kirby->request()->get('page')
-						]);
+						$collection = new UsersCollection(
+							users: $users->all()
+						);
```

Comment:
`Suggestion:` Might make it easier to understand what is what if we rather use `$component` here

```suggestion
						$component = new UsersCollection(
							users: $users->all()
						);
```